File: api/hospital-system/src/main/java/com/api/hospitalsystem/model/covidStatistics/StatisticResponse.java
package com.api.hospitalsystem.model.covidStatistics;

import lombok.Data;

import java.util.Date;

@Data
public class StatisticResponse {

    private String continent;
    private String country;
    private Date day;
    private Long population;
    private Date time;
    private StatisticCase cases;
    private StatisticDeaths deaths;
    private StatisticTests tests;

}

File: app/hospital-system/src/app/module/models/src/covid-statistics/statistic-response.ts
import { StatisticCase } from './statistic-case';
import { StatisticDeaths } from './statistics-deaths';
import { StatisticTests } from './statistics-tests';

export class StatisticResponse {
  continent: string = '';
  country: string = '';
  day: Date = new Date();
  population: number = 0;
  time: Date = new Date();
  cases: StatisticCase = new StatisticCase();
  deaths: StatisticDeaths = new StatisticDeaths();
  tests: StatisticTests = new StatisticTests();
}
